app/views/admins/index.html.slim

```diff
@@ -0,0 +1,22 @@
+h1 Listing admins
```

Comment:
do we have feature tests for this admin page logic?
do we need links to this page?

Reply:
> do we need links to this page?

added link to admins index which can only be used if signed in